packages/matchbox/src/components/Panel/styles.js

```diff
@@ -0,0 +1,97 @@
+import { tokens } from '@sparkpost/design-tokens';
+
+export const header = () => `
+  display: flex;
+  font-weight: ${tokens.fontWeight_semibold};
+  padding: ${tokens.spacing_500} ${tokens.spacing_500} 0 ${tokens.spacing_500};
+`;
+
+export const headerText = () => `
+  flex: 1 0 0;
+  padding-right: ${tokens.spacing_100};
+  font-size: ${tokens.fontSize_400};
+  font-weight: ${tokens.fontWeight_semibold};
+  line-height: ${tokens.lineHeight_300};
+  color: ${tokens.color_gray_900};
+  margin: 0;
+
+  text-rendering: optimizeLegibility;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+`;
+
+export const panel = () => `
+  position: relative;
+  margin-bottom: ${tokens.spacing_400};
+  background: ${tokens.color_white};
+`;
+
+export const panelInner = props => {
+  return `
+    border: ${tokens.borderWidth_100} solid ${tokens.color_gray_400};
+    border-top-width: ${props.accent ? '0' : '1px'};
+  `;
+};
+
+export const left = () => `
```

Comment:
Not sure we need these styles here, can use `<Box textAlign='left'>`. I think in general we should only use styled-components if we need dynamic styles